Lib/io.py

```diff
@@ -56,7 +56,21 @@
                  BufferedWriter, BufferedRWPair, BufferedRandom,
                  IncrementalNewlineDecoder, text_encoding, TextIOWrapper)
 
-OpenWrapper = _io.open # for compatibility with _pyio
+
+def __getattr__(name):
+    if name == "OpenWrapper":
+        import warnings
+        warnings.warn('OpenWrapper is deprecated, use open instead',
+                      DeprecationWarning, stacklevel=2)
+        # bpo-43680: Until Python 3.9, _pyio.open was not a static method and
+        # builtins.open was set to OpenWrapper to not become a bound method
+        # when set to a class variable. _io.open is a built-in function whereas
+        # _pyio.open is a Python function. In Python 3.10, _pyio.open() is now
+        # a static method, and builtins.open() is now io.open().
+        OpenWrapper = _io.open
```

Comment:
`global OpenWrapper` ?

Reply:
Oh, thanks! That's why the warning was emitted multiple times when using `-Walways` when io.OpenWrapper was read multiple times, now it makes sense! It's now fixed. I also wrote unit tests.